docs/source/guides/writer/chapters/basics.rst

```diff
@@ -69,3 +72,21 @@ currently exports to exec-test tests in default:
 +-----------------------------+---------------------------------------+-----------------------------------------------------------------------------------------------------+
 | `***`                       | All variables from --mux-yaml         | TIMEOUT=60; IO_WORKERS=10; VM_BYTES=512M; ...                                                       |
 +-----------------------------+---------------------------------------+-----------------------------------------------------------------------------------------------------+
+
+User's environment variables
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You can also let avocado set your own environment variables. For that, you need
+to pass the environment variables ass kwargs for the test. Here is an example
```

Comment:
s/ass/as/

Also, the info about kwargs is not universal, it's only about exec-test (and similars like tap).  So, I'd write:

"pass the environment variables as keyword arguments (``kwargs``) to the ``exec-test``s (and derived like ``tap``)".